src/translations/screens/subscreens/Login.ts

```diff
@@ -49,22 +49,23 @@ const LoginTexts = {
     title: _('Betingelser', 'Terms', 'Vilkår'),
   },
   activeFareContractPrompt: {
+    header: _('Logg inn', 'Log in', 'Logg inn'),
     title: _(
-      'Vent litt, du har en gyldig billett',
-      'Wait a sec – you have a valid ticket',
-      'Vent litt, du har ein gyldig billett',
+      'Billetten din slettes ved innlogging',
+      'Your ticket will be deleted upon logging in',
+      'Billetten din slettes ved innlogging',
     ),
     body: _(
-      'Billetten kan **ikke** overføres til din profil og vil slettes ved innlogging. Dersom du trenger billetten, vent til den er utløpt **før** du logger inn.',
-      'This ticket can **not** be transferred to your profile, and will be deleted upon login. If you need the ticket, wait until it has expired **before** logging in.',
-      'Billetten kan **ikkje** overførast til din profil og vil slettast ved innlogging. Dersom du treng billetten, vent til den er gått ut **før** du loggar inn.',
+      'Billetten din kan ikke overføres til en profil, fordi den er kjøpt anonymt. Vent med å logge inn til billetten utløper, for å unngå at den blir slettet. Slettede billetter **refunderes ikke.**',
+      'Your ticket cannot be transferred to a profile because it was purchased anonymously. Please wait until it expires before you log in, to avoid deletion. Deleted tickets are **non-refundable.**',
+      'Billetten din kan ikkje overførast til ein profil, sidan den vart kjøpt anonymt. Vent med å logge inn til billetten går ut, for å unngå at den blir sletta. Sletta billettar blir **ikkje refunderte.**',
```

Comment:
```suggestion
      'Billetten din kan ikkje overførast til ei profil, sidan han vart kjøpt anonymt. Vent med å logge inn til billetten går ut, for å unngå at han blir sletta. Sletta billettar blir **ikkje refunderte.**',
```

Reply:
Not sure about this suggestion?
https://ordbokene.no/nn/search?q=profil&scope=ei&perPage=20

I think it is fine as is.

Reply:
Agree with keeping ein, but what about han vs. den? @marius-at-atb / @rosvik

Reply:
Huh, I was sure it was "ei profil", but you're right 😅 I still think we should use "han", since "billetten" is hankjønn. https://www.sprakradet.no/svardatabase/sporsmal-og-svar/den-i-nynorsk-for-hanho/

Reply:
Since both are allowed and `den` was suggested by designers through sketches in Figma, we keep it like this ⛵